eventdata/schedulers/utilization_scheduler.py

```diff
@@ -22,57 +22,59 @@
 
 
 class UtilizationBasedScheduler:
+    RESPONSE_TIMES = []
     """
-    This scheduler schedules events at 100% utilization (unthrottled) during the warmup time-period. It tracks this
-    period itself (i.e. independently of Rally) using the task parameter ``warmup-time-period``. During this period
-    it gathers response time metrics. The median response time and the provided target utilization (via the task
+    This scheduler schedules events at 100% utilization (unthrottled) if is is in recording mode (enabled by setting 
+    ``record-response-times`` to ``True``). The median response time and the provided target utilization (via the task
     parameter ``target-utilization``) determine the average waiting time during the actual measurement phase of the
     benchmark. In order to avoid that clients coordinate, we randomize waiting time using a Poisson distribution.
```

Comment:
As discussed offline we'll change this to:

> To prevent clients from coordinating (i.e. executing requests at exactly the same time), we randomize waiting time using a Poisson distribution.